homeassistant/components/mqtt/config_integration.py

```diff
@@ -148,12 +152,37 @@
     required=True,
 )
 
+CONFIG_SCHEMA_ENTRY = vol.Schema(
+    {
+        vol.Optional(CONF_CLIENT_ID): cv.string,
+        vol.Optional(CONF_KEEPALIVE): vol.All(vol.Coerce(int), vol.Range(min=15)),
+        vol.Optional(CONF_BROKER): cv.string,
+        vol.Optional(CONF_PORT): cv.port,
+        vol.Optional(CONF_USERNAME): cv.string,
+        vol.Optional(CONF_PASSWORD): cv.string,
+        vol.Optional(SET_CA_CERT): vol.Any("off", "auto", "custom"),
```

Comment:
I believe there is no need to store SET_CA_CERT and SET_CLIENT_CERT. I will filter those out before updating the entry.